node/resolvers/document/index.ts

```diff
@@ -91,6 +107,36 @@ export const mutations = {
     }
   },
 
+  createDocumentV2: async (
+    _: any,
+    args: CreateDocumentV2Args,
+    context: Context
+  ) => {
+    const {
+      dataEntity,
+      document: { document },
+      schema,
+    } = args
+
+    const {
+      clients: { masterdata },
+    } = context
+
+    const response = (await masterdata.createDocument(
```

Comment:
If something goes wrong, what would happen? 🤔

Reply:
There is an error handler for each request, it checks the error code and raises an exception, then apollo takes care of returning the error.